.github/workflows/release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - uses: actions/checkout@v4
     - uses: actions/setup-python@v4
       with:
-        python-version: "3.x"
+        python-version: ">=3.9"
```

Comment:
Q: curious why this needed to change?  Just conscious that the release pipeline changes are untested here.

Reply:
This was because the env setup failed on 3.8, looking into it I don't think newest polars supports 3.8 https://pypi.org/project/polars/